### PR TITLE
Add a giphy search to chat

### PR DESCRIPTION
**Pull request by itsmaggiemay**

### CODE REVIEW

File: packages/client/src/components/Group/components/Chat/Chat.tsx [hunk 1]
```diff
@@ -164,6 +166,19 @@ export default function Chat(props: Props) {
     });
   };
 
+  const onGiphyPickerExited = () => {
+    // HACK: doesn't focus right without it ¯\_(ツ)_/¯
```
Comment (orrybaram) [inline packages/client/src/components/Group/components/Chat/Chat.tsx:170]: looks like the setTimeout is commented out, do we still need it?

File: packages/client/src/components/Group/components/Chat/components/GiphySearch/GiphySearch.tsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
+import 'emoji-mart/css/emoji-mart.css';
```
Comment (orrybaram) [inline packages/client/src/components/Group/components/Chat/components/GiphySearch/GiphySearch.tsx:1]: do we need this here?

File: packages/client/src/components/Group/components/Chat/Chat.tsx (hunk 1 above)
Comment (itsmaggiemay) [reply packages/client/src/components/Group/components/Chat/Chat.tsx:170]: nope I can remove it should I remove it

File: packages/client/src/components/Group/components/Chat/components/GiphySearch/GiphySearch.tsx (hunk 2 above)
Comment (itsmaggiemay) [reply packages/client/src/components/Group/components/Chat/components/GiphySearch/GiphySearch.tsx:1]: oops this one slipped in thanks!

File: packages/client/src/components/Group/components/Chat/Chat.tsx (hunk 1 above)
Comment (orrybaram) [reply packages/client/src/components/Group/components/Chat/Chat.tsx:170]: yeah, if we dont need it might as well pop it out

**Pull request merged**